externallib.php

```diff
@@ -33,7 +36,7 @@ public static function do_ai_magic($contextid, $jsondata)
 
             $attempt_record = self::getAIAttemptRecord($assignmentid, $userid);
 
-            if (self::isExceedingAttemptCount($attempt_record)) {
+            if (false) {
```

Comment:
This is never being run?

Reply:
``externallib.php`` is not used by anything. (Might be deprecate in the future as well)
I have remove it in the upcoming commit.